front/context/package.json

```diff
@@ -17,7 +17,12 @@
     "start": "yarn build --watch",
     "test": "echo skipped"
   },
+  "dependencies": {
+    "@substrate/local-storage": "^2.2.9"
+  },
   "peerDependencies": {
+    "@polkadot/api": "^1.9.1",
+    "@polkadot/extension-inject": "^0.21.1",
```

Comment:
if these are `peerDependencies`, then the gatsby app should install these as `dependencies` right?

Reply:
Yes, I made sure it's already the case.

Reply:
> is it enough to check it gets put into the root node_modules

Yes, node_modules is enough. Gatsby has `extension-dapp`, and `extension-dapp` automatically installs `extension-inject`